src/algorithms/digi/PhotoMultiplierHitDigi.cc

```diff
@@ -131,13 +131,13 @@ eicrecon::PhotoMultiplierHitDigi::AlgorithmProcess(std::vector<const edm4hep::Si
               m_log->trace("hit_group: pixel id={:#018x} -> npe={} signal={:<5g} time={:<5g}", id, hit.npe, hit.signal, hit.time);
 
         // build raw hits
-        std::vector<edm4eic::RawPMTHit*> raw_hits;
+        std::vector<edm4eic::RawTrackerHit*> raw_hits;
         for (auto &it : hit_groups) {
             for (auto &data : it.second) {
-                edm4eic::RawPMTHit* hit = new edm4eic::RawPMTHit{
+                edm4eic::RawTrackerHit* hit = new edm4eic::RawTrackerHit{
                   it.first,
-                  static_cast<uint32_t>(data.signal),
-                  static_cast<uint32_t>(data.time/m_cfg.timeStep)
+                  static_cast<int32_t>(data.signal),
+                  static_cast<int32_t>(data.time/m_cfg.timeStep)
```

Comment:
:-/ this should probably be `uint32_t` in EDM4eic (tag @sly2j). It doesn't make sense for these data words to be signed.

It will just cause issues. If you want to be resilient (and explicitly screw up at the boundaries):
```suggestion
                  static_cast<decltype(hit.charge)>(data.signal),
                  static_cast<decltype(hit.timeStamp)>(data.time/m_cfg.timeStep)
```
(untested)

Reply:
Maybe clearer if:
```suggestion
                  static_cast<decltype(edm4eic::RawTrackerHit::charge)>(data.signal),
                  static_cast<decltype(edm4eic::RawTrackerHit::timeStamp)>(data.time/m_cfg.timeStep)
```

Reply:
Definitely, actually this is what I'm doing in #393 already, but I will do it here now.